Mu2eUtilities/inc/EventWeightHelper.hh

```diff
@@ -10,14 +10,22 @@
 #ifndef Mu2eUtilities_inc_EventWeightHelper_hh
 #define Mu2eUtilities_inc_EventWeightHelper_hh
 
-#include <string>
-
-#include "canvas/Utilities/InputTag.h"
-#include "art_root_io/TFileDirectory.h"
-
-namespace fhicl { class ParameterSet; }
-namespace art { class Event; }
+#include <string>                       // for string
+#include <algorithm>                    // for max
+#include <vector>                       // for vector
+
+#include "canvas/Utilities/InputTag.h"  // for InputTag
+
+namespace fhicl {
+class ParameterSet;
+}  // namespace fhicl
+namespace art {
+class Event;
+}  // namespace art
```

Comment:
I am not a fan of closing brace comments when the braces are only a few lines apart.  Is there an option for a minimum span?

What code made the change from single line to  multi-line forward declarations?  iwyu or something else?

Reply:
one of IWYU's options is to put it on a multiline. I'm not sure why it put this as the default, but I can change this back to single line with an extra flag.